src/utils/ip.ts

```diff
@@ -40,3 +44,17 @@ export default function getIp(): string {
 
   return address
 }
+
+export function getPublicIp() {
+  axios({
+    method: 'GET',
+    url: 'https://ifconfig.co/ip',
+  })
+    .then((res) => {
+      log.info(`Monika is running on Public IP ${res?.data}`)
+      publicIpAddress = res?.data
```

Comment:
i think you should check first if res?.data has value before setting it to publicIpAddress and calling log.info

Reply:
fixed